assets/js/components/OutfrontTakeoverTool/OutfrontTakeoverTool.tsx

```diff
@@ -56,14 +67,27 @@ class OutfrontTakeoverTool extends React.Component<
         confirmJSX: <></>,
         onSubmit: this.toggleModal,
       },
+      stationScreenOrientationList: {
+        red: [],
+        orange: [],
+        blue: [],
+        silver: [],
+        green: [],
```

Comment:
What's the reason we have to specify this both here and on L171?

Reply:
There isn't one. I think this is leftover from when I was passing this state down via `props` instead of using the context. It will still need a default of `{}`, but no need to provide the full object shape in it.